packages/core/src/app/shipping/MultiShippingFormV2.scss

```diff
@@ -8,10 +8,154 @@
 }
 
 .consignment-header {
-    font-size: fontSize("large");
+    font-size: fontSize("small");
     margin: 0 0 spacing("single");
 }
 
 .add-consignment-button {
     color: color("greys", "dark");
 }
+
+.consignment-line-item-list {
+    list-style: none;
+    margin-left: 0;
+    li {
+        font-size: fontSize("smallest");
+        line-height: lineHeight("large");
+        .line-item-options {
+            color: color("greys");
+        }
+    }
+}
+
+.new-consignment-line-item-header, .consignment-line-item-header {
+    display: flex;
+    h3 {
+        margin-bottom: spacing("base");
+        font-size: fontSize("base");
+    }
+    p {
+        margin: 0;
+    }
+    a {
+        margin-left: spacing("base");
+        font-size: fontSize("base");
+    }
+}
+
+.consignment-line-item-header {
+    justify-content: space-between;
+    div {
+        display: flex;
+        .expand-items-button {
+            display: flex;
+            svg {
+                fill: currentColor;
+            }
+        }
+    }
+}
+
+.allocate-items-modal {
+    .modal-body {
+        padding-top: 0;
+
+        h3 {
+            margin-bottom: #{spacing("single") + spacing("half")};
+            font-weight: fontWeight("normal");
+        }
+    }
+
+    .modal-footer {
+        button {
+            margin-left: 0;
+            border: none;
+        }
+    }
+
+    .left-to-allocate-items-table-actions {
+        display: flex;
+        margin-bottom: spacing("base");
+        justify-content: space-between;
+        
+        p {
+            margin-bottom: 0;
+        }
+
+        div {
+            display: flex;
+            gap: spacing("half");
+        }
+    }
+
+    .left-to-allocate-items-table {
+        margin-top: spacing("base");
+        border-left: none;
+        border-right: none;
+    
+        thead {
+            background: none;
+            tr {
+                border-top: container("border");
+                border-bottom: container("border");
+                th:first-child {
+                    width: 50%;
```

Comment:
In order to manage all these numbers easily in the furture, can we create some variables and put them in one place? Maybe at top of this file?
https://github.com/bigcommerce/checkout-js/blob/c21be1be1b29d589e6473490112601ec469d8887/packages/core/src/scss/settings/global/layout/_layout.scss#L13